frog/imports/api/sessions.js

```diff
@@ -185,13 +185,18 @@ const addSessionFn = (graphId: string, slug: string): string => {
 
     const sessionId = uuid();
     const graph = Graphs.findOne(graphId);
-    const count = Graphs.find({
-      name: { $regex: '#' + graph.name + '*' }
-    }).count();
-    const sessionName = '#' + graph.name + ' ' + (count + 1);
-
+    const match = graph.name.match(/(.+)\(([^(]+)\)$/);
```

Comment:
Should be `/(.+)\(([0-9]+)\)$/`

If I name a graph "hello (world)" the session associated becomes "#hello (NaN)"